frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -128,19 +136,20 @@ function EmbedFrame({
               />
             )}
             {hasParameters && (
-              <div className="flex">
+              <ParametersWidgetContainer>
                 <SyncedParametersList
                   className="mt1"
                   question={question}
                   dashboard={dashboard}
                   parameters={getValuePopulatedParameters(
                     parameters,
-                    parameterValues,
+                    draftParameterValues ?? parameterValues,
```

Comment:
Repeat the same change I made for app dashboards, but this is for public dashboards and signed dashboard emdeddings.